src/main/groovy/Foo.groovy

```diff
@@ -26,7 +26,7 @@ class Foo {
             case 0: println "Zero"; break
             case 1: println "One"; break
             case 2: println "Two"; break
-            case 2: println "Another two, hm ...."; break
+            case 2: println "Another two, hm ..."; break
```

Comment:
[CodeNarc] WARNING: DuplicateCaseStatement: Duplicate case statements found in switch. org.codehaus.groovy.ast.stmt.CaseStatement@7a7471ce[expression: ConstantExpression[2]; code: org.codehaus.groovy.ast.stmt.BlockStatement@28276e50[org.codehaus.groovy.ast.stmt.ExpressionStatement@62e70ea3[expression:org.codehaus.groovy.ast.expr.MethodCallExpression@162be91c[object: org.codehaus.groovy.ast.expr.VariableExpression@3efe7086[variable: this] method: ConstantExpression[println] arguments: org.codehaus.groovy.ast.expr.ArgumentListExpression@675d8c96[ConstantExpression[Another two, hm ...]]]], org.codehaus.groovy.ast.stmt.BreakStatement@741b3bc3]]?.expression?.value appears twice